apps/etterlatte-saksbehandling-ui/client/src/components/behandling/trygdetid/OverstyrtTrygdetid.tsx

```diff
@@ -41,19 +44,44 @@ export const OverstyrtTrygdetid = ({
       )}
       {!redigerbar && overstyrtNorskPoengaar !== undefined && (
         <>
-          <BodyShort>
-            Eksportvurdering - pensjonen er innvilget etter unntaksregelen om at avdød har mindre enn 20 års botid,
-            {opptjentTekst()}:
-          </BodyShort>
-          <AntallAar>{trygdetid.overstyrtNorskPoengaar}</AntallAar>
+          {
+            {
+              [SakType.BARNEPENSJON]: (
+                <BodyShort>
+                  Eksportvurdering - pensjonen er innvilget etter unntaksregelen om at avdød har mindre enn 20 års
+                  botid,
+                  {opptjentTekst()}:
+                </BodyShort>
+              ),
+              [SakType.OMSTILLINGSSTOENAD]: (
+                <BodyShort>
+                  Ved eksport skal trygdetid være lik avdødes antall poengår fordi samlet botid til avdøde eller
+                  gjenlevende er mindre enn 20 år:
+                </BodyShort>
+              ),
+            }[sakType]
```

Comment:
Funky stuff

Reply:
Inline switch er nice, her går det fint men man skal være litt forsiktig med bruken da alt "rendres" uavhengig state. Litt usikker på bruken her. Men akkurat for disse tilfellene så er en ternary mer enn nok

Reply:
Hva er egentlig fordelen med denne kontra en IIFE? altså {() => switch()..}

Reply:
Usikker på om jeg er helt enig at ternary er nok, det gir jo ingen utfyllende sjekk av at alle tilstander i enumen er sjekket? Bare antar at hvis det ikke er feks barnepensjon så må det være omstillingsstønad. Se for deg hvor moro det er å nøste opp i den koden når PO'et finner ut at vi skal håndtere en tredje ytelse 😛 Da liker jeg bedre å kunne endre enumet, så får man beskjed om alle plasser hvor det må fikses. Mulig varianten jeg bruker er litt i overkant hacky og at en switch er bedre i disse tilfellene?